src/main/java/com/listywave/list/application/domain/CategoryType.java

```diff
@@ -8,24 +11,35 @@
 @AllArgsConstructor
 public enum CategoryType {
 
-    ENTIRE("1","ENTIRE","전체"),
-    CULTURE("2","CULTURE","문화"),
-    LIFE("3","LIFE","일상생활"),
-    PLACE("4","PLACE","장소"),
-    MUSIC("5","MUSIC","음악"),
-    MOVIE_DRAMA("6","MOVIE_DRAMA","영화/드라마"),
-    BOOK("7","BOOK","도서"),
-    ANIMAL_PLANT("8","ANIMAL_PLANT","동식물"),
-    ETC("9","ETC","기타"),
+    ENTIRE("0","전체"),
+    CULTURE("1","문화"),
+    LIFE("2","일상생활"),
+    PLACE("3","장소"),
+    MUSIC("4","음악"),
+    MOVIE_DRAMA("5","영화/드라마"),
+    BOOK("6","도서"),
+    ANIMAL_PLANT("7","동식물"),
+    ETC("8","기타"),
     ;
 
     private final String codeValue;
-    private final String nameValue;
     private final String korNameValue;
 
     public static CategoryType enumOf(String codeValue) {
         return Arrays.stream(CategoryType.values())
                 .filter( t -> t.getCodeValue().equals(codeValue))
-                .findAny().orElse(null);
+                .findAny().orElseThrow(() -> new CustomException(ErrorCode.NOT_FOUND, "해당 카테고리코드는 존재하지 않습니다."));
     }
+
+    @JsonCreator
+    public static CategoryType fromString(String key) {
+        for(CategoryType category : CategoryType.values()) {
+            if(category.name().equalsIgnoreCase(key)) {
+                return category;
+            }
+        }
+        throw new CustomException(ErrorCode.NOT_FOUND, "해당 카테고리는 존재하지 않습니다.");
+    }
+
+
```

Comment:
```suggestion
return Arrays.stream(CategoryType.values())
        .filter(categoryType -> categoryType.name().equalsIgnoreCase(key))
        .findFirst()
        .orElseThrow(() -> new CustomException(ErrorCode.NOT_FOUND, "에러 메시지"));
```
stream 쓰면 코드가 더 눈에 확 들어올 것 같아요